triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -393,26 +393,33 @@ object Server {
             server.logTriggerStatus(runningTrigger, "running")
             server.addRunningTrigger(runningTrigger) match {
               case Left(err) =>
+                // The trigger has just advised it's in the running
+                // state but updating the running trigger table has
+                // failed. This error condition is exogenous to the
+                // runner. We therefore need to tell it explicitly to
+                // stop.
                 val msg = "Failed to add running trigger to database.\n" + err
                 ctx.self ! TriggerInitializationFailure(runningTrigger, msg)
```

Comment:
I'm not sure sending this message makes sense anymore. Previously we wanted this case to be handled in the same way as other initialization failures, but now we are restarting those. So I think we can just log the status instead of going through another indirection.